packages/chip/src/lib/chip.stories.mdx

```diff
@@ -0,0 +1,349 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import pkg from '../../package.json';
+import ChairOutlinedIcon from '@mui/icons-material/ChairOutlined';
+import Chip from './chip';
+
+<Meta title="Chip" component={Chip} />
+
+# Chip <img src={`https://img.shields.io/badge/Version-v${pkg.version}-brightgreenn`} />
+
+### Installation
+
+First install the latest version of the package from `@novatics/chip`.
+
+```shell
+  yarn add -D @novatics/chip
+```
+
+### Basic Usage
+
+The `Chip` component have two variants: it supports outlined and filled styling. The default variant is filled.
+
+<Canvas>
+  <Story name="Basic Usage">
+    {(args) => {
+      return (
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'row',
+            alignItems: 'center',
+            gap: '10px',
+          }}
+        >
+          <Chip label="Chip" />
+          <Chip label="Chip" variant="outlined" />
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+```tsx
+import Chip from '@novatics/chip';
+
+<Chip label="Chip" />
+<Chip label="Chip" variant="outlined" />
+```
+
+### Click interaction
+
+Chips can be clickable. In this case, it will have a hover and a click effect. It can be set by either passing the `clickable` prop or by passing a function to `onClick` prop.
+
+<Canvas>
+  <Story name="Click interaction">
+    {(args) => {
+      return (
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'row',
+            alignItems: 'center',
+            gap: '10px',
+          }}
+        >
+          <Chip label="Clickable Chip" clickable />
+          <Chip
+            label="Click Me"
+            onClick={() => window.alert('Hi! :)')}
+            variant="outlined"
+          />
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+```tsx
+<Chip label="Clickable Chip" clickable />
```

Comment:
no need to add sample of code. When we use canvas, storybook provides a "show code" option
![image](https://user-images.githubusercontent.com/15728617/230217709-d3344557-0298-4291-bf18-ab7a68a67de9.png)